holobot/extensions/dev/workflows/show_available_servers_workflow.py

```diff
@@ -56,7 +56,18 @@ async def change_page(
         state: Any
     ) -> InteractionResponse:
         if not isinstance(context, ServerChatInteractionContext):
-            return InteractionResponse(EditMessageAction(content="This interaction is available in a server only."))
+            return InteractionResponse(
+                action=EditMessageAction(
+                    content="This interaction is available in a server only."
+                )
+            )
+
+        if not isinstance(state, PagerState):
```

Comment:
Modified this file because I found out it was type unsafe due to the `if isinstance` happening after the usage.